Source/WebKit/Shared/WebEvent.h

```diff
@@ -108,4 +108,39 @@ class WebEvent {
     WallTime m_timestamp;
 };
 
+WTF::TextStream& operator<<(WTF::TextStream&, WebEvent::Type);
+
 } // namespace WebKit
+
+namespace WTF {
+
+template <> struct EnumTraits<WebKit::WebEvent::Type> {
```

Comment:
I don't think this is still needed with Alex's new serialization.in approach?

Reply:
We are trying to migrate all EnumTraits to *.serialization.in files.  Since this is already in WebEvent.serialization.in, this is completely unneeded.

Reply:
It doesn't build without it.